src/ui/public/vislib/visualizations/_map.js

```diff
@@ -259,16 +259,23 @@ export default function MapFactory(Private, tilemap, $sanitize) {
         if (NWlng < -180) {
           NWlng += 360;
         }
+
+        const SELat = bounds.getSouthEast().lat;
+        const NWLat = bounds.getNorthWest().lat;
+        if (SELat === NWLat || SElng === NWlng) {
+          return;
```

Comment:
+1 for (1).

Let's pass on (2) :P. NW/SE/N/W/SSW/... are canonical acronyms for the cardinal directions. Spelling it out makes it longer for no reason afaic.